src/Base/FileTemplate.cpp

```diff
@@ -23,10 +23,6 @@
 
 #include "PreCompiled.h"
 
-#ifndef _PreComp_

```

Comment:
FileTemplate.cpp serves as template for newly created files to see what's needed. Thus the PreComp stuff shouldn't be removed.